extension/reearth-yml.ts

```diff
@@ -11,21 +11,26 @@ const yml = {
   version: pkg.version,
   extensions: [
     {
-      id: "search-widget",
+      id: "toolbar-widget",
       type: "widget",
-      name: "search",
+      name: "toolbar",
```

Comment:
Widget names should be capitalized

Reply:
Fixed! [fix widget name](https://github.com/eukarya-inc/PLATEAU-VIEW-3.0/pull/14/commits/af62634a3e0cb3cbc76a3fef1182421006fc84f5)